### PR TITLE
Increase patch test coverage

### PR DESCRIPTION
**Pull request by dstansby**
Increase test coverage of `patches.py`

### CODE REVIEW

**Comment by dstansby** [comment]
Ideally I'd like to cover `ConnectionPatch`s too, but #9518 is stopping me at the moment.

File: lib/matplotlib/tests/test_patches.py [hunk 1]
```diff
@@ -350,3 +350,15 @@ def test_units_rectangle():
     ax.add_patch(p)
     ax.set_xlim([4*U.km, 7*U.km])
     ax.set_ylim([5*U.km, 9*U.km])
+
+
+@image_comparison(baseline_images=['connection_patch'], extensions=['png'],
```
Comment (tacaswell) [inline lib/matplotlib/tests/test_patches.py]: remove the text from this one too?

**Comment by Kojoley** [comment]
ping

**Comment by dstansby** [comment]
Text removed. Ready to go once tests all pass!

**Pull request merged**